source/NVDAObjects/IAccessible/ia2Web.py

```diff
@@ -49,9 +50,28 @@ class Application(Document):
 class BlockQuote(Ia2Web):
 	role = controlTypes.ROLE_BLOCKQUOTE
 
-class Editor(Ia2Web):
+class Editor(Ia2Web, DocumentWithTableNavigation):
 	TextInfo = MozillaCompoundTextInfo
 
+	def _getTableCellAt(self,tableID,startPos,destRow,destCol):
+		obj=startPos.NVDAObjectAtStart
+		while not obj.table and obj!=self:
+			obj=obj.parent
+		if not obj.table:
+			raise LookupError
+		table = obj.table
+		try:
+			try:
+				cell = table.IAccessibleTable2Object.cellAt(destRow - 1, destCol - 1).QueryInterface(IAccessibleHandler.IAccessible2)
+			except AttributeError:
+				cell = table.IAccessibleTableObject.accessibleAt(destRow - 1, destCol - 1).QueryInterface(IAccessibleHandler.IAccessible2)

```

Comment:
is this guaranteed to succeed?

Reply:
It is guaranteed not to throw AttributeError. But it may throw COMError. The reason for catching AttributeError above is that IAccessibleTable2Object may not exist. However, at least one of IAccessibleTable2Object or IAccessibleTableObject will exist.

Reply:
Maybe it's assumed knowledge, but could you add a comment here to point that out?